src/ServiceControl.Monitoring/MetricReport.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace NServiceBus.Metrics
+{
+    using Newtonsoft.Json.Linq;
+
+    /// <summary>
+    /// The reporting message.
+    /// </summary>
+    public class MetricReport : ICommand
```

Comment:
So we assume here that for now we ignore the potential extensions, but potentially will use it in the future?